tests/python/mkl/test_mkldnn.py

```diff
@@ -405,14 +420,18 @@ def check_pooling_training(stype):
             in_location = [mx.nd.array(data_tmp).tostype(stype)]
 
             if np.array(shape).shape[0] == 3:
-                test = mx.symbol.Pooling(data=data, kernel=(3), stride=(2), pool_type='avg')
+                test = mx.symbol.Pooling(data=data, kernel=(
+                    3), stride=(2), pool_type='avg')
             elif np.array(shape).shape[0] == 4:
-                test = mx.symbol.Pooling(data=data, kernel=(3, 3), stride=(2, 2), pool_type='avg')
+                test = mx.symbol.Pooling(data=data, kernel=(
+                    3, 3), stride=(2, 2), pool_type='avg')
```

Comment:
It looks strange when part of the parameter is moved to the next line. "(3, 3)" shouldn't' be spitted

Reply:
Indeed, max limit was added max-line-length = 120. Should be fine right now. Please have a look at that.